src/components/Main/DiscordInvite/DiscordInvite.tsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
-
-// TODO: Add text and link
-// TODO: Add graphic at the end of the page
+import {Row, Col} from 'react-bootstrap';
 
 function DiscordInvite() {
   return (
-    <div>
-      <h4>Want to join Girls Who Code at UVA?</h4>
-      <a href="#" target="_blank"><h2>Join our Discord server (arrow)</h2></a>
-      <img src="#" alt="Orange wave saying 'Get Started'" />
-    </div>
+      <Row>
+        <Col sm={12}>
+          <h4 className="text-turq pl-lg-5 ml-lg-5 pl-md-5 ml-md-5 pl-sm-5 ml-sm-5">Want to join Girls Who Code at UVA?</h4>
```

Comment:
you don't need pl-lg-5 or md, etc. if you do pl-sm-5, it'll be the padding for small and up. if you want it to be the same no matter the size, try out pl-5 and just skip the size